features/edition-images.feature

```diff
@@ -19,4 +19,15 @@ Feature: Images tab on edit edition
     And I have the "Preview images update" permission
     And a draft document with images exists
     When I visit the images tab of the the document with images
-    Then I should see a list of the images
\ No newline at end of file
+    Then I should see a list with 2 images
+
+  Scenario: Images can be deleted from the images tab
+    Given I am a writer
+    And I have the "Preview images update" permission
+    And a draft document with images exists
+    When I visit the images tab of the the document with images
```

Comment:
Actually, sorry, one final thing. I've spotted a typo in this step name.

I know you didn't introduce it, but would you mind fixing it before merging this?

```suggestion
    When I visit the images tab of the document with images
```